### DISCUSSION TITLE
Full screen behavior of chrome with Wayland backend

### DISCUSSION THREAD

**Discussion by Quatih** [hyprwm/Hyprland#2386] [Q&A]
If the Preferred Ozone platform flag in chromium is set to Wayland, triggering full screen in any way will full screen the same way as F11 does, hiding the tabs. Is it possible to force behavior like when it's using Xwayland, to maximize the window? I tried the different window rules but they don't seem to change anything.
The X11 option is blurry with mixed dpi monitors.

**Comment by riley-martin** [comment]
> If the Preferred Ozone platform flag in chromium is set to Wayland, triggering full screen in any way will full screen the same way as F11 does, hiding the tabs. Is it possible to force behavior like when it's using Xwayland, to maximize the window?   

Maybe using the `fullscreen, 1` dispatcher instead of `fullscreen, 0`.

**Reply by Quatih** [reply]
Thanks, that could be the fix I need, but ironbar still stays in the foreground. Could make a script to work around it.
